internal/ingress/controller/kong.go

```diff
@@ -114,6 +119,91 @@ func (n *NGINXController) OnUpdate(ingressCfg *ingress.Configuration) error {
 	return nil
 }
 
+func (n *NGINXController) syncGlobalPlugins() error {
+	glog.Infof("syncing global plugins")
+
+	targetGlobalPlugins, err := n.store.ListGlobalKongPlugins()
+	if err != nil {
+		return err
+	}
+	targetPluginMap := make(map[string]*pluginv1.KongPlugin)
+	var duplicates []string // keep track of duplicate
+
+	for i := 0; i < len(targetGlobalPlugins); i++ {
+		name := targetGlobalPlugins[i].Name
+		// empty name skip it
+		if name == "" {
+			continue
+		}
+		if _, ok := targetPluginMap[name]; ok {
+			glog.Error("Multiple KongPlugin definitions found with 'global' annotation for :", name,
+				", the plugin will not be applied")
```

Comment:
Since the behavior here is to not load all plugins with the same name (as in L147-150, if i understand if correctly), should we also put that into the error log (and further more in the doc)?
Something like `Multiple KongPlugin definitions with same name found with 'global' annotation, all plugin defined with name '", name, "' will not be loaded`

Reply:
I think the current error message makes sense as it conveys the essence of the error.
I'll be putting in docs later in a separate PR.